GENESIS.md

```diff
@@ -31,11 +28,24 @@ dep ensure
 go run main.go
 ```
 
-A few lines of text should be printed, followed by two lines listing the total address count and total ATOM supply, and `penultimate_genesis.json` should be written.
+A few lines of text should be printed, followed by two lines listing the total address count and total ATOM supply, and `genesis.json` should be written.
```

Comment:
In all other places this is still called `penultimate_genesis.json`?

Reply:
Yeh I left it in the ceremony documentation. Should I just recommit the old penultimate back so the links work?

Reply:
Ok, I see. 

> Should I just recommit the old penultimate back so the links work?

I guess it might be a little confusing otherwise.

Reply:
Ok pushed itback